### PR TITLE
Interpreter global opcode

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

**Review by robert-zaremba — APPROVED** [review]

File: packages/algorand-js/test/fixture-projects/teal-files/test-global.teal [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+global MinBalance
+global MaxTxnLife
+global ZeroAddress
+global GroupSize
+global LogicSigVersion
+global Round
+global LatestTimestamp
+global CurrentApplicationID
+
+
```
Comment (amityadav0) [inline packages/algorand-js/test/fixture-projects/teal-files/test-global.teal:13]:
```suggestion
```

File: packages/algorand-js/src/interpreter/opcode-list.ts [hunk 2]
```diff
@@ -1241,6 +1241,50 @@ export class Return extends Op {
   }
 }
 
+// push field from current transaction to stack
+export class Global extends Op {
+  readonly field: string;
+  readonly interpreter: Interpreter;
+
+  /**
+   * @param args words list extracted from line: {global field (string)}
```
Comment (amityadav0) [inline packages/algorand-js/src/interpreter/opcode-list.ts:1250]: Please update doc string

File: packages/algorand-js/src/interpreter/opcode-list.ts [hunk 3]
```diff
@@ -1241,6 +1241,50 @@ export class Return extends Op {
...
+   * @param line line number in TEAL file
+   * @param interpreter interpreter object
+   */
+  constructor (args: string[], line: number, interpreter: Interpreter) {
+    super();
+    assertLen(args.length, 1, line);
+    this.assertGlobalDefined(args[0]);
+
+    this.field = args[0]; // field
```
Comment (amityadav0) [inline packages/algorand-js/src/interpreter/opcode-list.ts:1259]:
```suggestion
    this.field = args[0]; // global field
```

File: packages/algorand-js/src/interpreter/opcode-list.ts [hunk 4]
```diff
@@ -1241,6 +1241,50 @@ export class Return extends Op {
...
+    this.interpreter = interpreter;
+  }
+
+  execute (stack: TEALStack): void {
+    let g;
```
Comment (amityadav0) [inline packages/algorand-js/src/interpreter/opcode-list.ts:1264]: can we rename this to `result` or something more descriptive?

File: packages/algorand-js/src/interpreter/opcode-list.ts (hunk 2 above)
Comment (ratik21) [reply packages/algorand-js/src/interpreter/opcode-list.ts:1250]: yes

File: packages/algorand-js/src/interpreter/opcode-list.ts (hunk 4 above)
Comment (ratik21) [reply packages/algorand-js/src/interpreter/opcode-list.ts:1264]: renaming to `globalField`

**Pull request merged**